final/MealsharingNET/Models/User.cs

```diff
@@ -0,0 +1,9 @@
+namespace MealsharingNET;
```

Comment:
The namespace here should contain `Models`, as the class is in the Models folder